src/angular/styles/_typography.scss

```diff
@@ -1547,19 +1547,6 @@ textarea {
   &:not(:disabled):not(.sbb-disabled) {
     cursor: pointer;
   }
-
```

Comment:
This was a misunderstanding. Only the lines that you added in this PR should be removed.